### ISSUE TITLE
Bitgo create wallet endpoint connection timeout response

### ISSUE THREAD

**Issue by winni4eva** [BitGo/BitGoJS#506]
I am running bitgoJs using the docker image built from the docker file
```
docker run -e BITGO_USE_PROXY="socks://1.1.1.1" -p 3080:3080 e83107097df1
BitGo-Express running
Environment: test
Base URI: http://0.0.0.0:3080
```
Then I make a request to create a wallet
```
POST: http://0.0.0.0:3080/api/v2/tbtc/wallet/generate
data : {
  "label": "mylabel",
  "passphrase": "mypass"
}
headers: Authorization: Bearer xxxxx
Content-Type: application/json
```
Always Returns Response 
```
{"error":"Connection Timed Out","message":"Connection Timed Out"}
```

**Comment by tylerlevine** [comment]
Is there a socks proxy listening at `1.1.1.1`? Isn't that cloudflare's DNS IP? Unless I'm mistaken, I don't think they have a socks proxy listening there.

**Comment by winni4eva** [comment]
@tylerlevine Thanks for pointing me out the cause. I just typed the the 1.1.1.1 proxy ip when i was creating the issue. I actually picked one from the internet which does not seem to be working well from your answer. After starting the express server without any proxy 
```
./bitgo-express --debug --port 3080 --env test --bind localhost
```
requests to endpoint
```
http://127.0.0.1:3080/api/v1/client/constants
```
now succeed, which was also timing out when i had a proxy set. I will try and get a working proxy ip and try again.
